components/Layout.js

```diff
@@ -143,7 +143,12 @@ export default class Layout extends React.Component {
                   className="nv-s-l-a"
                   href="https://data.boston.gov/dataset/vision-zero-crash-records"
                 >
-                  Download the data
+                  Crash data
+                </a>
+              </li>
+              <li className="nv-s-l-i">
+                <a className="nv-s-l-a" href="">
```

Comment:
`href`?

Reply:
waiting for the fatality dataset to be published on Analyze Boston to link to here. Don't have a link yet since it's not published, added a comment above this line